requirements/edx/base.txt

```diff
@@ -185,7 +185,7 @@ pycryptodomex==3.9.8      # via -r requirements/edx/base.in, edx-proctoring, lti
 pygments==2.6.1           # via -r requirements/edx/base.in
 pyjwkest==1.4.2           # via -r requirements/edx/base.in, edx-drf-extensions, lti-consumer-xblock
 pyjwt[crypto]==1.7.1      # via -r requirements/edx/base.in, drf-jwt, edx-rest-api-client, social-auth-core
-pymongo==3.10.1           # via -r requirements/edx/base.in, -r requirements/edx/paver.txt, edx-opaque-keys, event-tracking, mongodbproxy, mongoengine
+pymongo==3.11.0           # via -r requirements/edx/base.in, -r requirements/edx/paver.txt, edx-opaque-keys, event-tracking, mongodbproxy, mongoengine
```

Comment:
If you want to leave this update in here, I would mention it in the commit message, since it's a pretty core, low-level dependency.  FWIW - here's the release notes for pymongo 3.11.0: https://pymongo.readthedocs.io/en/3.11.0/changelog.html

Reply:
interesting, this looks quite low level indeed so don't want to introduce this risk thanks, will pull this one out (just edit this line in base.txt or use constraints?)

Reply:
Just pull it out in `base.txt`  and the other requirements `.txt` files, don't constrain it.  I pinged the arch team to help assess level of risk and potential mitigations.

Reply:
got it, thank you!